go.mod

```
@@ -0,0 +1,61 @@
+module github.com/ONSdigital/dp-maps-api
+
+go 1.17
+
```

If you run 'go mod tidy' then it will add something like this:

```suggestion
require (
	github.com/ONSdigital/dp-component-test v0.6.5
	github.com/ONSdigital/dp-healthcheck v1.2.3
	github.com/ONSdigital/dp-net v1.4.1
	github.com/ONSdigital/log.go/v2 v2.1.0
	github.com/cucumber/godog v0.10.0
	github.com/gorilla/mux v1.8.0
	github.com/kelseyhightower/envconfig v1.4.0
	github.com/pkg/errors v0.9.1
	github.com/smartystreets/goconvey v1.7.2
	github.com/stretchr/testify v1.7.0
)
```

---

updated